vendor/ethereum_support/src/erc20_quantity.rs

```diff
@@ -1,38 +1,52 @@
-use crate::{u256_ext::ToBigDecimal, web3::types::Address, U256};
+use crate::{
```

Comment:
❓ what is the reasoning behind the split?

Reply:
https://github.com/comit-network/comit-rs/pull/695/commits/b38007f9aa23cefcdc64a4fe0c2adad76af2e345